src/main/java/de/tum/in/www1/artemis/domain/Submission.java

```diff
@@ -76,6 +78,10 @@ public abstract class Submission extends DomainObject implements Comparable<Subm
     @Column(name = "submission_date")
     private ZonedDateTime submissionDate;
 
+    @Column(name = "plagiarism_detected")
+    @JsonView(QuizView.Before.class)
+    private Boolean plagiarismDetected;
```

Comment:
Needed to get this back to `Boolean` as this primitive type was causing 500 errors for already existing submissions

Reply:
cc: @JohannesStoehr

Reply:
You can also adapt the changes to insert a default value for existing entries